### PR TITLE
SPLICE-2086: CDH 5.14.0 support (master)

### PR DESCRIPTION
**Pull request by perryluopl**
Merge cdh 5.14.0 support to master branch.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1015,7 +1015,34 @@
...
+            <properties>
+                <cdh.parcel.version>5.14.0</cdh.parcel.version>
+                <envClassifier>cdh5.14.0</envClassifier>
+                <envHbase>hbase1.2.0</envHbase>
+                <hadoop.version>2.6.0-cdh5.14.0</hadoop.version>
+                <hbase.version>1.2.0-cdh5.14.0</hbase.version>
+                <hive.version>1.1.0-cdh5.14.0</hive.version>
+                <spark.major.version>spark2.2</spark.major.version>
+                <zookeeper.version>3.4.5-cdh5.14.0</zookeeper.version>
+                <spark.version>2.2.0.cloudera1</spark.version>
```
Comment (OlegMazurov) [inline pom.xml]: Should be consistent with other cdh platforms: 2.2.0.cloudera2

File: pom.xml [hunk 2]
```diff
@@ -1015,7 +1015,34 @@
...
+                <spark-assembly-id>spark-assembly-hadoop2.6.0-cdh5.14.0</spark-assembly-id>
+                <kafka.version>0.9.0-kafka-2.0.2</kafka.version>
```
Comment (OlegMazurov) [inline pom.xml]: Ditto: 0.10.2-kafka-2.2.0

File: platforms/cdh5.14.0/pom.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  ~ If not, see <http://www.gnu.org/licenses/>.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>splice_machine_cdh5.14.0</artifactId>
+    <packaging>pom</packaging>
+    <parent>
+        <artifactId>spliceengine-parent</artifactId>
+        <groupId>com.splicemachine</groupId>
+        <version>2.5.0.1808-SNAPSHOT</version>
```
Comment (OlegMazurov) [inline platforms/cdh5.14.0/pom.xml]: Wrong version.

File: pom.xml [hunk 4]
```diff
@@ -1040,6 +1040,34 @@
...
+                <envClassifier>cdh5.14.0</envClassifier>
+                <envHbase>hbase1.2.0</envHbase>
+                <hadoop.version>2.6.0-cdh5.14.0</hadoop.version>
+                <hbase.version>1.2.0-cdh5.14.0</hbase.version>
+                <hive.version>1.1.0-cdh5.14.0</hive.version>
+                <spark.major.version>spark2.2</spark.major.version>
+                <zookeeper.version>3.4.5-cdh5.14.0</zookeeper.version>
+                <spark.version>2.2.0.cloudera2</spark.version>
+                <spark-assembly-id>spark-assembly-hadoop2.6.0-cdh5.14.0</spark-assembly-id>
+                <kafka.version>0.9.0-kafka-2.0.2</kafka.version>
```
Comment (OlegMazurov) [inline pom.xml]: Use 0.10.2-kafka-2.2.0 for consistency.

File: platforms/cdh5.14.0/pom.xml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        <artifactId>spliceengine-parent</artifactId>
+        <groupId>com.splicemachine</groupId>
+        <version>2.8.0.1810-SNAPSHOT</version>
+    </parent>
+    <profiles>
+        <profile>
+            <id>ee</id>
+            <activation>
+                <property>
+                    <name>enterpise</name>
```
Comment (wtang16) [inline platforms/cdh5.14.0/pom.xml]: Please fix spelling before merging.

File: platforms/cdh5.14.0/pom.xml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                    <value>ee</value>
+                </property>
+            </activation>
+            <properties>
+                <enterpise>ee</enterpise>
```
Comment (wtang16) [inline platforms/cdh5.14.0/pom.xml]: Same here

File: platforms/cdh5.14.0/pom.xml (hunk 5 above)
Comment (OlegMazurov) [reply platforms/cdh5.14.0/pom.xml]: That would be a massive change. The wrong spelling permeates many pom files. Affects 2.5/2.7 as well. Worth a separate JIRA.

**Comment by wtang16** [comment]
https://splice.atlassian.net/browse/SPLICE-2100 is created to follow up.

**Pull request merged**